src/track/serato/markers2.cpp

```diff
@@ -2,6 +2,23 @@
 
 #include <QtEndian>
 
+namespace {
+QString zeroTerminatedUtf8StringtoQString(QDataStream* stream) {
+    DEBUG_ASSERT(stream);
+
+    QByteArray data;
+    quint8 byte = '\xFF';
```

Comment:
Maybe you can avoid some casting: '\xFF' are of type char with is implementation defined either qint8 or quint8. 
I guess in most cases QDataStream will automatically do it right without casting.